src/guidescanpy/flask/blueprints/job_query.py

```diff
@@ -71,6 +71,7 @@ def result(format, job_id):
                                 hit["off-target-summary"],
                                 hit["cutting-efficiency"],
                                 hit["specificity"],
+                                hit["gc-content"],
```

Comment:
If you add a row to the csv file, won't the header go out of sync? I think we need to modify the header as well.